keyboards/dztech/dz65rgb/keymaps/fronti90/keymap.c

```diff
@@ -0,0 +1,74 @@
+#include QMK_KEYBOARD_H
+
+#define DEFAULT_LAYER 0
+#define GAMING_LAYER 1
+#define FN_LAYER 2
+
+unsigned char r_orange=0xFF;
+unsigned char g_orange=0x80;
+unsigned char b_orange=0x00;
+
+unsigned char r_green=0x44;
+unsigned char g_green=0xaa;
+unsigned char b_green=0x77;
```

Comment:
Just a heads up, there is a prefined list of colors: 
https://github.com/qmk/qmk_firmware/blob/master/quantum/color.h

And this is included automatically.  Is probably better than adding 6 unsigned char to handle that.